generate-release/src/release_notes.rs

```diff
@@ -107,3 +127,81 @@ file_name = "{file_name}.md"
+    // If it does, we don't want to spam the repo with duplicate issues
+    if existing_issue_titles.contains(&issue_title) {
+        println!("Issue already exists for PR #{}: {}", pr_number, title);
+        return;
+    }
+
+    let pr_url = format!("https://github.com/bevyengine/bevy/pull/{pr_number}",);
+    let file_path = file_path.to_string_lossy();
+
+    // The weird indentation is intentional.
+    // Otherwise, the text is not formatted correctly on GitHub as the tabs are copied into the issue body.
+    let issue_body = format!(
+        "{pr_url} needs release notes for the upcoming Bevy release!
+
+Please reply below if you'd like to write these notes. 
+While the author(s) of the PR often have the context, knowledge and motivation to draft the release notes for their feature, anyone can contribute release notes!
+
+------
+
+Release notes should:
+
+1. Clearly motivate the change.
+2. Be written in a way that is understandable by the average Bevy user: some programming background and a general understanding of games.
+3. Show off the coolest features of the PR. Screenshots are awesome, but elegant APIs are also welcome!
+4. If this was a perf-centric PR, quantify the performance improvements. Graphs and statistics work well for this.
+
+We can help you revise the release notes: a rough draft alone is incredibly useful :)
+Your expertise is invaluable for contextualizing the changes; we'll work with you to bring the technical writing up to par.
+
+To submit your release notes, modify `{file_path}` and submit a PR.
+In that PR, please mention this issue with the `Fixes #ISSUE_NUMBER` keyphrase so it gets closed automatically.");
+
+    let labels = vec!["A-Release-Notes", "C-Content", "S-Ready-For-Implementation"];
+
+    if dry_run {
+        println!("Would open issue on GitHub:");
+        println!("Title: {}", issue_title);
+        println!("Body: {}", issue_body);
+        println!("Labels: {:?}", labels);
+    } else {
+        // Open an issue on the `bevy-website` repo
+        let response = client
+            .open_issue("bevy-website", &issue_title, &issue_body, labels)
+            .unwrap();
+        println!("Opened issue for PR #{}: {}", pr_number, title);
+        // Pause between opening issues to avoid getting rate-limited.
+        // See https://docs.github.com/en/rest/using-the-rest-api/best-practices-for-using-the-rest-api?apiVersion=2022-11-28#pause-between-mutative-requests
+        std::thread::sleep(std::time::Duration::from_secs(2));
+
+        // Leave a comment on the PR linking to the new issue
+        let issue_url = response.html_url;
+
+        let comment = format!("Thank you to everyone involved with the authoring or reviewing of this PR! This work is relatively important and needs release notes! Head over to {issue_url} if you'd like to help out.",);
+
+        // Unwrap to warn the user if the comment fails
+        client.leave_comment("bevy", pr_number, &comment).unwrap();
```

Comment:
Can we avoid the `unwrap()` and instead log an error to whoever is running that command? Management scripts crashing is a plague of software maintenance 😅

Reply:
Can do!